app/questionnaire/navigator.py

```diff
@@ -101,6 +101,11 @@ def __init__(self, survey_json, metadata=None, answer_store=None):
         self.metadata = metadata or {}
         self.survey_json = survey_json
 
+        self.preceeding_path = []
```

Comment:
I wonder if an approach like the interstitial blocks may work? i.e., define the introduction as a block? that would mean no custom logic here? also it could allow us to have multiple introductions pages for example. just thinking possibly more flexibility and a cleaner schema?